packages/flame/lib/src/effects/glow_effect.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:ui';
+
+import 'package:flame/components.dart';
+import 'package:flame/effects.dart';
+
+class GlowEffect extends ComponentEffect<HasPaint> {
+  GlowEffect(this.style, this.blurValue, super.controller, {this.paintId});
+
+  final BlurStyle style;
+  final double blurValue;
```

Comment:
For a user, it would be kinda confusing to have a `GlowEffect` with a `blurValue`. Why not call it `glowStrength` or something like that?

Reply:
renamed to `strength`